### PR TITLE
Run nodejs tests with multiple nodejs versions

### PR DESCRIPTION
**Pull request by Patrik-Stas**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -292,24 +290,33 @@ jobs:
   test-node-wrapper:
     needs: workflow-setup
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [12.x, 18.x]
```
Comment (mirgee) [inline .github/workflows/main.yml:295]: What is the issue with node wrapper unit tests on node 19? The integration tests seem to run fine on node 19.

Comment (Patrik-Stas) [reply .github/workflows/main.yml:295]: Had some issues .... I'll try to add it again

**Comment by mirgee** [comment]
Running tests for these node versions might seem to suggest that they are "supported", while in fact the slowdown makes the library unusable with a node version > 12.

**Comment by Patrik-Stas** [comment]
@mirgee we might wait another 2 years for that issue to be resolved ^_^ It's open since 2020

**Comment by mirgee** [comment]
@Patrik-Stas Let's merge it - we can undo once it starts causing issues, and at least we will know which change started to cause them.

**Review by mirgee — APPROVED** [review]

**Pull request merged**